libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -688,21 +688,24 @@ pub fn ballot_card(
         })
         .unwrap_or_default();
 
+    let normalized_front_image = imageproc::contrast::threshold(&front_image, front_threshold);
+    let normalized_back_image = imageproc::contrast::threshold(&back_image, back_threshold);
+
```

Comment:
FYI, I think we should do this in place instead of creating a clone ([PR](https://github.com/votingworks/vxsuite/pull/6523)).